services/src/pro/layers/postgres_layer_db.rs

```diff
@@ -794,20 +796,76 @@ where
 
         let stmt = conn
             .prepare(
-                "
-               SELECT 
-                   definition
-               FROM 
-                   layer_providers
-               WHERE
-                   id = $1",
+                "SELECT
+                    definition, NULL AS pro_definition
```

Comment:
couldn't you use coalesce here and return a single column?

Reply:
Unfortunately not, they are different types.

Reply:
Unfortunately not, they are distinct column types.